src/styles/components/layout/buttons/TertiaryButton/_TertiaryButton.scss

```
@@ -34,4 +34,13 @@
 
   background: none;
   border: none;
+
+  &:disabled {
+    color: $grey-dark;
```

```suggestion
    color: $grey-medium;
```
D'après cette maquette https://app.zeplin.io/project/5e05ce53fdf73e9a53ed33e9/screen/5faeb54637b7cf63cc8565d6